common-library/src/main/java/com/sdevelopment/skt/common/domain/Product.java

```diff
@@ -0,0 +1,57 @@
+package com.sdevelopment.skt.common.domain;
+
+import org.hibernate.validator.constraints.NotBlank;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "products")
+//@Data
+@NamedStoredProcedureQueries({
+        @NamedStoredProcedureQuery(name = "saveProduct",
+                procedureName = "insertproductsp",
+                parameters = {
+                        @StoredProcedureParameter(mode = ParameterMode.IN, name = "_name", type = String.class)
+                }),
+
+        @NamedStoredProcedureQuery(name = "getAllProducts",
+                procedureName = "listproductssp",
+                resultClasses = Product.class)
+})
+public class Product implements Serializable {
+    @Id
+    @Column(name = "id")
+    private Integer id;
+
+    @Column(name = "name")
+    //@NotEmpty(message = "Name may not be empty")
+    @NotBlank(message = "Name may not be empty")
+    private String name;
```

Comment:
if the attribute name is the same as the column name, it's not necessary to use the `@Column` annotation